public/static/locales/en/metadata.json

```diff
@@ -10,6 +10,8 @@
     "attrTypeLabel": "Type",
     "avus": "AVUs",
     "back": "Back",
+    "confirmDiscardChangesDialogHeader": "Discard Changes?",
+    "confirmDiscardChangesDialogMsg": "You have unsaved changes. Do you wish to continue without saving?",
```

Comment:
When I trying this PR locally, I ran into confusion here with the generic `Ok` and `Cancel` action button. We should change this to prompt the user for `saving` the changes rather than `discarding`. 
`"confirmSaveChangesDialogHeader": "Save Changes?",
"confirmSaveChangesDialogMsg": "You have unsaved changes. Do you wish to save now?"`
and the dialog options should be `Save`, `Discard` and `Cancel` rather than generic `OK` and `Cancel`.

Reply:
I don't know if we are doing this consistently throughout Sonora, if not we should start doing this (i.e providing specific actions instead of generic `OK` and `Cancel`).

Reply:
I seem to remember having this discussion before where we actually reached the opposite conclusion. I think in the old DE we had 3 buttons like that for some dialogs, but we decided it was too much cognitive load on the user, and we decided on only 1 or 2 buttons for cancel/confirm or cancel/ok instead. I think this was also a recommendation from the Material spec. I will see if I can find a reference to that discussion again.

Reply:
Ahh I do remember now something about having 3 buttons. But I still recommending having actions specific buttons instead of generic `Ok / Cancel`.

Reply:
I couldn't find where we had that discussion, but I did find on the Material spec they recommend a max of 2 buttons: https://material.io/components/dialogs#actions (under the `Number of actions` section).

I also agree the button text could be more specific 👍
That's recommended by the Material spec as well (https://material.io/components/dialogs#alert-dialog).

I think we should stick with the `Cancel` / `Discard` buttons, though, because this confirmation dialog only shows when the user is trying to close the template without saving, so we're confirming that choice. Otherwise they should `Cancel` the discard action and specifically select the `Save` button (which is an action that can add or modify many AVUs at a time).

Reply:
Sounds good to me 👍🏽

Reply:
I also updated the content text, so the `Discard` button with dialog's message is hopefully clearer:
`You have unsaved changes. Do you wish to continue and discard these changes?`